v2.1/admin-ui-statements-page.md

```diff
@@ -0,0 +1,110 @@
+Statement | The SQL statement or the fingerprint of similar SQL statements.
+Time | The cumulative time taken to execute the SQL statement (or multiple statements having the same fingerprint).
+Count | The total number of times the SQL statement (or multiple statements having the same fingerprint) is executed. <br><br>The execution count is displayed in numerical value as well as in the form of a horizontal bar. The bar is color-coded to indicate the ratio of runtime success (indicated by blue) to runtime failure (indicated by red) of the execution count for the fingerprint. The bar also helps you compare the execution count across all SQL fingerprints in the table. <br><br>You can sort the table by count.
+Mean Rows | The average number of rows returned or affected while executing the SQL statement (or multiple statements having the same fingerprint). <br><br>The number of mean rows is displayed in numerical value as well as in the form of a horizontal bar. The bar helps you compare the mean rows across all SQL fingerprints in the table. <br><br>You can sort the table by mean rows.
+Mean Latency | The average service latency of the SQL statement (or multiple statements having the same fingerprint). <br><br> The mean latency is displayed in numerical value as well as in the form of a horizontal bar. The bar is color-coded to indicate the latency across the execution phases: parse (indicated by red), plan (indicated by yellow), execute (indicated by blue), and overhead (indicated by red). The bar also helps you compare the mean latencies across all SQL fingerprints in the table. <br><br>You can sort the table by mean latency.
+
+To view additional details of a statement fingerprint, click on the statement fingerprint in the **Statement** column to see the **Statement Details** page.
+
+## Statement Details page
+
+The **Statement Details** page displays the details of the execution count, latency by phase, row count, and statistics for the selected statement fingerprint.
+
+### Execution count
+
+The **Execution Count** table provides information about the following parameters in numerical values as well as bar graphs:
+
+Parameter | Description
+-----|------------
+First Attempts | The cumulative number of first attempts to execute the SQL statement (or multiple statements having the same fingerprint).
+Retries | The cumulative number of retries to execute the SQL statement (or multiple statements having the same fingerprint).
+Max Retries |
+Total |
+
+### Latency by phase
+
+The **Latency by Phase** table provides the mean and standard deviation values of the service latency for each execution phase (parse, plan, run, and overhead) for the SQL statement (or multiple statements having the same fingerprint). The table provides the service latency details in numerical values as well as bar graphs. The bar graphs are color-coded per execution phase:
+
+Phase | Color code
+-----|------------
+Parse | Red
+Plan | Yellow
+Run | Blue
+Overhead | Red
```

Comment:
There's also a row for "Overall", which in the alpha has the numbers but no bar.

Reply:
Done.